boot/boot.js

```diff
@@ -2674,6 +2674,20 @@ $tw.hooks.addHook = function(hookName,definition) {
 	}
 };
 
+/*
+Delete hooks from the hashmap
+*/
+$tw.hooks.removeHook = function(hookName,definition) {
+	if($tw.utils.hop($tw.hooks.names,hookName)) {
+		var index = $tw.hooks.names[hookName].findIndex(function(hook) {
```

Comment:
We can't use findIndex() in the core

Reply:
Ok, I think

```js
$tw.hooks.names[hookName] = $tw.hooks.names[hookName].filter(function(hook) {
  return hook !== definition;
});
```

does the same, but I use the way identical to `removeEventListener`